src/models/projectConfiguration.ts

```diff
@@ -1,9 +1,10 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Why do you need this. Please remove it if you don't have a good reason as it will reduce the code safety for code written in this file.

If you actually need it prefer to suppress the specific line, instead of suppressing the entire file.
```ts
// eslint-disable-next-line no-unused-vars
const unusedVariable = 'I will not be flagged by ESLint';
```